src/filesystem.cpp

```diff
@@ -309,44 +310,47 @@ AsyncIOContext *FileAccess::asyncfopen(string *f)
 
 bool FileAccess::asyncopenf()
 {
+    numAsyncReads++;
```

Comment:
this counter registers the number of attempts, regardless the success or failure